config/config_test.go

```diff
@@ -30,6 +30,16 @@ func TestParse(t *testing.T) {
 			{"/sig/ssh-user-cert", []string{"key3"}, 36000},
 			{"/sig/blob", []string{"key1"}, 36000},
 		},
+		ShutdownOnFrequentSigningFailure: true,
+		ShutdownOnSigningFailureCriteria: struct {
+			ConsecutiveCountLimit uint
+			TimerDurationSecond   uint
+			TimerCountLimit       uint
```

Comment:
Maybe we could use struct embedding and use named struct instead of an anonymous one, so we don't need to repeat the struct definition here?